paddle/fluid/framework/data_type_test.cc

```diff
@@ -38,3 +38,25 @@ TEST(DataType, float16) {
   std::string type = "::paddle::platform::float16";
   EXPECT_STREQ(f::DataTypeToString(dtype).c_str(), type.c_str());
 }
+
+TEST(DataType, bfloat16) {
```

Comment:
The first test could be parameterized and reused here, to avoid duplicating of code.
This can be done later as a refactoring.

Reply:
You are right. I will refactor it later.